backend/fms_core/templates.py

```diff
@@ -119,6 +120,21 @@
       ("Library", "Sample Container Coordinates", "coordinates"),],
 }
 
+SAMPLE_ADD_METADATA_TEMPLATE = {
```

Comment:
I would remove the _ADD in the name of the template it does not only add metadata.